src/vs/workbench/contrib/chat/browser/chatAccessibilityService.ts

```diff
@@ -25,10 +32,14 @@ export class ChatAccessibilityService extends Disposable implements IChatAccessi
 
 	private _requestId: number = 0;
 
+	private readonly notifications: Set<DisposableStore> = new Set();
```

Comment:
I thought maybe we had a `DisposableSet`, but we only have `DisposableMap`. `DisposableSet` would be a very helpful utility for this kind of scenario though to avoid having to handle disposing and removing items manually!